src/main.ts

```diff
@@ -686,6 +687,15 @@ export class Purchases {
     return `${ANONYMOUS_PREFIX}${uuid.replace(/-/g, "")}`;
   }
 
+  /**
+   * Track an event.
+   * @param props - The properties of the event.
+   * @internal
+   */
+  public _trackEvent(props: TrackEventProps): void {
```

Comment:
Do we also need to export `TrackEventProps` as an internal API?

Reply:
Good call. Just updated. Although afaik, given those are not exposed it should not have a side-effect.

Reply:
Related, but I'm not sure if the convention for interfaces is to also prefix it with an underscore like `_TrackEventProps`